test.js

```diff
@@ -89,7 +92,8 @@ const types = new Map([
 		Object.create(null),
 		new Object() // eslint-disable-line no-new-object
 	]],
-	['integer', 6]
+	['integer', 6],
+	['domElement', document.createElement('div')]
```

Comment:
What about more than a div?

you can do `['div', 'input', 'span', 'img', 'canvas', 'h1', 'h2', 'h3' ...].map(e => document.createElement(e))`

Reply:
Agreed, but we don't need the different `h` tags.

Reply:
Let's go with: `div`, `input`, `span`, `img`, `canvas`, `script`.

Reply:
Also, what markup object is not considered a domElement? So that we can check for more cases